src/plugins/vis_type_vega/public/services.ts

```diff
@@ -35,3 +35,5 @@ export const [getInjectedVars, setInjectedVars] = createGetterSetter<{
 }>('InjectedVars');
 
 export const getEnableExternalUrls = () => getInjectedVars().enableExternalUrls;
+
+export const [getDocLinks, setDocLinks] = createGetterSetter<DocLinksStart>('docLinks');
```

Comment:
Using `createGetterSetter` is a bit of an outdated pattern, but I have no objection to following the Vega pattern until we actually refactor this.

cc @alexwizp for refactoring the Vega setup code